tests/func/test_external_repo.py

```diff
@@ -1,24 +1,33 @@
 import os
+import time
 
 from mock import patch
 
 from dvc.external_repo import external_repo
 from dvc.scm.git import Git
+from dvc.compat import fspath
 from dvc.utils.fs import path_isin
 
 
-def test_external_repo(erepo):
-    url = erepo.root_dir
+def test_external_repo(erepo_dir):
+    with erepo_dir.chdir():
+        erepo_dir.dvc_gen("file", "master", commit="create file on master")
+        # FIXME just debugging
+        time.sleep(1)
```

Comment:
So here we go again. Mac on travis doesn't checkout in time, so I have to add this `sleep()` here. Probably funky APFS again, same as it was before in other places. Will take a closer look to see if there is an elegant solution for that.

Reply:
Ok, so looks like older versions of xcode didn't support high-resolution timestamps, so upgrading from 9.4 to 11.3 fixed the issue. It might've been fixed in a version before 11.3, but I don't really see a point in trying to pinpoint some specific version right now, as 11.3 works for us as well. So hopefully this won't ever happen again. :pray: